src/components/Ballot.tsx

```diff
@@ -10,32 +10,35 @@ import HelpPage from '../pages/HelpPage'
 import NotFoundPage from '../pages/NotFoundPage'
 import PreReviewPage from '../pages/PreReviewPage'
 import PrintPage from '../pages/PrintPage'
-import ResetPage from '../pages/ResetPage'
 import ReviewPage from '../pages/ReviewPage'
 import SettingsPage from '../pages/SettingsPage'
 import StartPage from '../pages/StartPage'
 
 const Ballot = () => {
-  const { election, contests } = useContext(BallotContext)
-
-  const { bmdConfig } = election!
-  const { requireActivation, showHelpPage, showSettingsPage } = bmdConfig!
+  const { ballotStyleId, contests, election, precinctId } = useContext(
+    BallotContext
+  )
+  const {
+    requireActivation,
+    showHelpPage,
+    showSettingsPage,
+  } = election!.bmdConfig!
 
   return (
     <Switch>
-      {requireActivation ? (
-        <Route exact path="/" component={ActivationPage} />
-      ) : (
+      {!requireActivation || (ballotStyleId && precinctId) ? (
```

Comment:
I'm having trouble following this logic, and it looks error prone. For safety, error checks should, IMO, be closer to where the functionality is happening.

Reply:
Pushing an update to make it more readable. 

(Though, as we really have no direct need for a config option to enable/disable activation, I'd prefer to just remove the config options 😄)